events/views.py

```diff
@@ -7,39 +8,49 @@
 from .serialisers import EventSerializer
 
 
-class EventViewSet(GenericViewSet,
-                   generics.ListAPIView,
-                   mixins.ListModelMixin,
-                   mixins.CreateModelMixin,
-                   generics.GenericAPIView,
-                   APIView,):
+class EventViewSet(
+    GenericViewSet,
+    generics.ListAPIView,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    generics.GenericAPIView,
+    APIView,
+):
+    """A class based view for Events"""
```

Comment:
Add more detailed docstring in here. This is what is displayed against the browseable API.

This docstring needs to explain to a new user what this endpoint is, what it is for and how they can use it. 
Include information about the query parameters and give examples of how to use them.


e.g.
http://api.catalogue.ceda.ac.uk/api/v2/observations/
https://github.com/cedadev/ceda_moles_django/blob/e59d8eb79ff07d993648c3e2b633465d91ccd1e4/cedamoles_api/v2/views/observation.py#L135

Reply:
I an unable to see the that repo, I would like to know how to add headings/convention to writing the class docstring, right now following pydoc's convention.

Reply:
![image](https://user-images.githubusercontent.com/22395703/128539080-59a91267-489f-4450-82cb-85e3fe30626b.png)

Reply:
it's just markdown